test/Interop/Cxx/static/inline-static-member-var-silgen.swift

```diff
@@ -0,0 +1,14 @@
+// Test that inline static member variables are handled properly.
+
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -I %S/Inputs -emit-sil -enable-cxx-interop -Xcc -std=c++17 %s -o %t/static-member-var.sil
+// RUN: %FileCheck < %t/static-member-var.sil %s
+
+// CHECK: sil hidden @$s4main16readStaticMembers5Int32VyF : $@convention(thin) () -> Int32
+// CHECK: %0 = global_addr @_ZN22WithInlineStaticMember12staticMemberE : $*Int32
```

Comment:
Do you want to also test the code generation pattern for the read and write into the variable?